src/app/[locale]/(app)/credits/_components/Contributors.tsx

```diff
@@ -0,0 +1,155 @@
+"use client";
+
+import Section from "~/components/Section";
+import contributorsData from "../../../../../../contributors.json";
+import DetailCard from "~/components/data/DetailCard";
+import {
+  BugAntIcon,
+  CommandLineIcon,
+  FilmIcon,
+} from "@heroicons/react/24/solid";
+import Flag from "./Flag";
+import { partitionArray } from "~/utils";
+import DiscordImage from "~/components/DiscordImage";
+
+type Contributor = {
+  avatarUrl: string;
+  name: string;
+  description?: string;
+  bugs?: number;
+  translation?: string;
+  developer?: number;
+  influencer?: boolean;
+  priority?: number;
+};
+
+// TODO: sort
+
+function sortContributors() {
+  const contributors: Contributor[] = [];
+  let remainingContributors: Contributor[] = contributorsData;
+
+  // Priority contributors
+  {
+    const [priorityContributors, rest] = partitionArray(
+      remainingContributors,
+      (c) => c.priority !== undefined
+    );
+    remainingContributors = rest;
+
+    priorityContributors.sort((a, b) => a.priority! - b.priority!);
+    contributors.push(...priorityContributors);
+  }
+
+  // Devs
+  {
+    const [devContributors, rest] = partitionArray(
+      remainingContributors,
+      (c) => c.developer !== undefined
+    );
+    remainingContributors = rest;
+
+    devContributors.sort((a, b) => a.developer! - b.developer!);
+    contributors.push(...devContributors);
+  }
+
+  // Bug hunters
+  {
+    const [bugContributors, rest] = partitionArray(
+      remainingContributors,
+      (c) => c.bugs !== undefined
+    );
+    remainingContributors = rest;
+
+    bugContributors.sort((a, b) => a.bugs! - b.bugs!);
+    contributors.push(...bugContributors);
+  }
+
+  // Influencers
+  {
+    const [influencers, rest] = partitionArray(
+      remainingContributors,
+      (c) => !!c.influencer
+    );
+    remainingContributors = rest;
+
+    contributors.push(...influencers);
+  }
+
+  // Translators
+  {
+    contributors.push(...remainingContributors);
+  }
+
+  return contributors;
+}
+
+const contributors = sortContributors();
+
+function ContributorCard({ contributor }: { contributor: Contributor }) {
+  const {
+    avatarUrl,
+    name,
+    description,
+    bugs = 0,
+    translation,
+    developer = 0,
+    influencer,
+  } = contributor;
+
+  return (
+    <DetailCard
+      title={name}
+      description={
+        description ||
+        `${
+          bugs + developer + (translation ? 1 : 0) + (influencer ? 1 : 0)
+        } contributions`
+      }
+      leftSlot={
+        <div className="relative aspect-square w-10">
+          <DiscordImage
```

Comment:
I'm not confident about this one. Not every contributor will want to put its Discord image. Maybe what we can do is check if the avatarUrl contains `https://discord.com` and load the right component. Tell me if that sounds good to you @Androz2091 and I'll implement

Reply:
I guess we can turn off conditional rendering inside `<DiscordImage/>` if the host does not match `discord.com`/`discordapp.com`, does that sound good?

Reply:
Yes @skdhg, that would be perfect

Reply:
Do you want to make a PR for it @skdhg?